.github/workflows/ci.yaml

```diff
@@ -235,6 +235,43 @@ jobs:
           echo "skip_coverage: ${skip_coverage}"
           echo "skip_coverage=${skip_coverage}" >> $GITHUB_OUTPUT
 
+  sync-github-labels:
+    name: Sync GitHub integration labels
+    runs-on: ubuntu-24.04
+    if: github.ref == 'refs/heads/dev'
+    needs:
+      - info
+    steps:
+      - name: Check out code from GitHub
+        uses: actions/checkout@v4.2.2
+      - name: Sync integration labels
+        run: |
+          gh_labels=$(gh api /repos/${OWNER}/${REPO}/labels \
+            --paginate \
+            --jq '.[] | select(.name | startswith("integration: ")) | "\(.name)|\(.description);"')
+
+          for integration_manifest in homeassistant/components/*/manifest.json; do
```

Comment:
I think we can use the generated file

Reply:
I tried, but the `homeassistant/generated/integrations.json` is not "plain" - it has sometimes nested integrations. I tried to just search for all dict objects, containing the `integration_type` but my `jq`-fu is limited :see_no_evil: